### DISCUSSION TITLE
ji-tang/info/C-X3Z7w3Ndt-N-6i3iLaJ

### DISCUSSION THREAD

**Discussion by giscus** [wurencaideli/dumogu-ypwz-comment#29] [Announcements]
# ji-tang/info/C-X3Z7w3Ndt-N-6i3iLaJ

不是只有视频和音乐才带给人欢乐！还有一篇文字哟！学生时代所看过的短文还有谁记得呢？甚是怀念呢！

https://www.dumogu.top/ji-tang/info/C-X3Z7w3Ndt-N-6i3iLaJ?isInfo=false

**Comment by wjh46** [comment]
沙发
